dashboard/README.md

```diff
@@ -0,0 +1,62 @@
+## Go Ethereum Dashboard
+### Description
+
+The dashboard is a data visualizer integrated into geth, intended to collect and visualize useful information of an Ethereum node. 
+The dashboard consists of two parts:
+* The server listens to connections, collects data with a given refresh rate, and updates the dashboards through the opened connections.
+* The client waits for update messages, updates the content and tries to reconnect on connection loss.
+
+### Users
+#### Installation steps
+
+1. `cd .../go-ethereum/`
+1. `go install -v ./cmd/geth`
+1. Run the server with `geth --rinkeby --dashboard --vmodule=dashboard=5 --metrics`.
+1. Enter `localhost:8080` (or change the configuration).
```

Comment:
No need for user docs, that may or may not be done in some global documentation.

Reply:
`--dashboard` should auto-enable `--metrics`.